x/staking/types/params.go

```diff
@@ -33,12 +33,27 @@ const (
 )
 
 var (
-	KeyUnbondingTime     = []byte("UnbondingTime")
-	KeyMaxValidators     = []byte("MaxValidators")
-	KeyMaxEntries        = []byte("MaxEntries")
-	KeyBondDenom         = []byte("BondDenom")
-	KeyHistoricalEntries = []byte("HistoricalEntries")
-	KeyPowerReduction    = []byte("PowerReduction")
+	// ValidatorBondFactor of -1 indicates that it's disabled
+	ValidatorBondDisabled = sdk.NewDecFromInt(sdk.NewInt(-1))
+
+	// DefaultValidatorBondFactor is set to -1 (disabled)
```

Comment:
you mean the decision to disable by default?

Reply:
yeah

Reply:
same response as other comment:
> I don't think it was that intentional - but at the time, the thinking was probably that it didn't make sense to use gaia's specific params here as defaults (since this was originally written to be used with other chains). It also aligns with the current mainnet right now which has no caps in place